### PR TITLE
Protractor

### PR DESCRIPTION
**Pull request by lamara**
Adds a protractor helper file for use in upper-level tests. Also has a fix for a bug where we weren't properly encoding URLs used in requests.

### CODE REVIEW

**Comment by dlongley** [comment]
LGTM, merge.

File: bedrock/test.js [hunk 1]
```diff
@@ -1,7 +0,0 @@
-/*
```
Comment (mattcollier) [inline bedrock/test.js:1]: This file did not get moved into `test` and therefore the mocha tests cannot be run.  This file needs to be resurrected and the `test` script in `package.json` should be updated to reflect the new location.

@dlongley do we want our convention to be to keep the mocha and protractor tests in separate folders as we are doing elsewhere?

Comment (dlongley) [inline bedrock/test.js:1]: We want to keep mocha and protractor tests separate, yes.

**Comment by mattcollier** [comment]
@lamara still need to update:
- https://github.com/digitalbazaar/bedrock-notifications/blob/protractor/lib/test.config.js#L8
- https://github.com/digitalbazaar/bedrock-notifications/blob/protractor/package.json#L7

**Pull request merged**